opcodes/arcxx-opc.inc

```diff
@@ -64,6 +64,19 @@ insert_rbb (unsigned long long  insn,
   return insn | ((value & 0x07) << 8) | (((value >> 3) & 0x07) << 1);
 }
 
+/* Insert RB register pair into a pushdl/popdl instruction.  */
+
+static unsigned long long
+insert_rbb_chk (unsigned long long  insn,
+	    long long value,
+	    const char **errmsg ATTRIBUTE_UNUSED)
```

Comment:
`errmsg` is not `ATTRIBUTE_UNUSED` anymore.

Reply:
Fixed, thank you!